### DISCUSSION TITLE
How to store old values

### DISCUSSION THREAD

**Discussion by usauerbrey** [edterbak/HeishaMoNR#102] [Q&A] [answered]
Currently in my configuration, all old values (e.g. last 31 days and last 24h, etc.) are lost after reboot.
What can I do to preserve them.

**Comment by edterbak** [comment, marked as answer]
You need to follow the instructions on how to get persistant data enabled. 
https://github.com/edterbak/NodeRed_Heishamon_control
See the colapsed section underneath [Installation]

After you THINK it is done and working, test if it is actually working.

Store some variables... wait 300+ seconds (=flush timer to disk)
reboot. 
Only if the variables like the WAR graph, or RTC settings stick, you have proven it is working
